src/uct/ib/rdmacm/rdmacm_ep.c

```diff
@@ -97,7 +97,7 @@ static UCS_CLASS_INIT_FUNC(uct_rdmacm_ep_t, uct_iface_t *tl_iface,
               "iface cm_id: %p remote addr: %s",
                iface, iface->event_ch, iface->cm_id,
                ucs_sockaddr_str((struct sockaddr *)sockaddr->addr,
-                                ip_str, ip_len));
+                                ip_port_str, ip_port_len));
```

Comment:
some optimization trick: what about replace ip_port_str by ucs_alloca(ip_port_len) call - in this case if ucs_debug is not enabled (expanded to empty statement) alloca will not affect program at all

Reply:
@yosefe what do you think about such dirty optimization?

Reply:
i think it's illegal. and compiler could optimize it anyway
http://man7.org/linux/man-pages/man3/alloca.3.html
```
    On many systems alloca() cannot be used inside the list of arguments
       of a function call, because the stack space reserved by alloca()
       would appear on the stack in the middle of the space for the function
       arguments.
```